hack/undeploy-ocp.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+oc get crds | grep leaderworkerset | awk '{print $1}' | xargs oc delete crd
```

Comment:
In my opinion this script can be simplified via;

```sh
oc delete -f deploy/
```

yes, it will fail while deleting 07_lws-operator.cr.yaml but that also proves that 07_lws-operator.cr.yaml is also deleted. We can catch the error and return with zero exit code.

Reply:
Actually, overall we don't need this undeploy-ocp.sh. Anyone who works on this repository can simply run this command `oc delete -f deploy/`

Reply:
Ok, About the negative test for uninstalling operator, we can cover it in future operator installation via catalogsource,  I delete the step, could you check again.

Reply:
Moved ARTIFACT_DIR part to https://github.com/openshift/lws-operator/pull/99, will close the pr.